.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: node_js
+node_js:
+- 10
+install:
+- npm install
+script:
+- npm run build
+- npm run build-prod
+deploy:
+  provider: pages
+  local_dir: src
+  skip_cleanup: true
+  github_token: $GITHUB_TOKEN
+  on:
+    branch: master
```

Comment:
https://docs.travis-ci.com/user/deployment/pages/ - the caveat at the top seems to imply that deploying from master is a bad idea (unless `keep-history` is true)